node/test/go/dev-mode/test/integration/index.test.js

```diff
@@ -81,6 +82,10 @@ describe('Integration', function () {
               // Since we are sending spans as they are complete I need to change this to a variable length
               // but we should always have at least 1
               expect(traces.length >= 1).to.equal(true);
+              // For whatever reason the traces are being written in a bad wire format? 🤔
+              // const traceData = Buffer.from(traces[0], 'base64');
+              // const tracePayload = traceProto.TracePayload.decode(traceData);
+              // expect(tracePayload.events.length).to.equal(2);
```

Comment:
@medikoo If you pull down this branch, uncomment lines `8` and `86:88`, and run these integration tests locally do you also get the wire format error I am seeing?

I know that is kind of a lot to ask but I am just trying to eliminate a local testing issue here 😅